src/snowflake/snowpark/_internal/analyzer/unary_plan_node.py

```diff
@@ -47,13 +85,37 @@ def __init__(
         self.grouping_expressions = grouping_expressions
         self.aggregate_expressions = aggregate_expressions
 
+    @property
+    def individual_complexity_stat(self) -> Counter[str]:
+        estimate = Counter()
+        if self.grouping_expressions:
+            # GROUP BY grouping_exprs
+            estimate += Counter({ComplexityStat.GROUP_BY.value: 1}) + sum(
+                (expr.cumulative_complexity_stat for expr in self.grouping_expressions),
+                Counter(),
+            )
+        else:
+            # LIMIT 1
+            estimate += Counter({ComplexityStat.LOW_IMPACT.value: 1})
+
+        get_complexity_stat = (
+            lambda expr: expr.cumulative_complexity_stat
```

Comment:
```suggestion
            getattr(expr, "cumulative_complexity_stat", Counter({ComplexityStat.COLUMN.value: 1}))
```
and no function is needed?

Reply:
yep. updated